go.mod

```diff
@@ -34,15 +34,14 @@ require (
 	golang.org/x/sys v0.16.0
 	golang.org/x/term v0.16.0
 	golang.zx2c4.com/wireguard v0.0.0-20230313165553-0ad14a89f5f9
-	google.golang.org/api v0.114.0
-	google.golang.org/grpc v1.56.3
+	google.golang.org/api v0.47.0
+	google.golang.org/grpc v1.52.0-dev
```

Comment:
Why was it downgraded?

Reply:
Um, I'm not sure. I've run this `go get github.com/NordSecurity/libtelio@...` command and it seems it changed it. I'll fix this

Reply:
I think I've fixed this